.travis.yml

```diff
@@ -37,14 +30,30 @@ jobs:
       env:
         global:
           - USE_TEST_UTILS2=false
+      before_script:
+        # Copy custom configs from the repo because PG-13 isn't set up to run like
+        # it normally does on Travis out of the box.
+        - sudo cp travis/postgresql.conf /etc/postgresql/13/main/postgresql.conf
+        - sudo cp travis/pg_hba.conf /etc/postgresql/13/main/pg_hba.conf
+        - sudo pg_ctlcluster 13 main restart
       script:
         - tox
-    - name: test utils2
+    - name: test utils2 noproc
```

Comment:
What is 'proc' versus 'noproc'?

Reply:
Proc means pytest-postgresql will spin up postgres process. 
Noproc means pytest-postgresql will use existing postgres instance.

Reply:
Can we document that in the file? Or another place capturing similar concepts?